### PR TITLE
Easier to read JSON file

### PR DESCRIPTION
**Pull request by profxj**
Adds a new method to writing jSON files which is easier to read.
Preserves the old methods as default

Adds a test

### CODE REVIEW

File: linetools/utils.py [hunk 1]
```diff
@@ -201,17 +201,44 @@ def jsonify(obj, debug=False):
     return obj
 
 
-def savejson(filename, obj, overwrite=False, indent=None):
-    """ Save a python object to filename using using the JSON encoder."""
+def savejson(filename, obj, overwrite=False, indent=None, easy_to_read=False,
+             **kwargs):
+    """ Save a python object to filename using using the JSON encoder.
```
Comment (ntejos) [inline linetools/utils.py]: delete one 'using'.

Comment (profxj) [inline linetools/utils.py]: fixed

**Comment by profxj** [comment]
Reading is fine and tested

**Comment by profxj** [comment]
merging

**Pull request merged**